core-blocks/more/test/__snapshots__/edit.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`core/more/edit should match snapshot when noTeaser is false 1`] = `
 <React.Fragment>
   <IfBlockEditSelected(InspectorControlsFill)>
     <PanelBody>
-      <_class
+      <WithInstanceId(ToggleControl)
```

Comment:
The nice part about this helper is that we no longer see those cryptic component names 👍

Reply:
Ya that's definitely useful :)